explore/visualization/data/sample_roc_data.csv

```diff
@@ -0,0 +1,25 @@
+false_positive,true_positive,curve,data

```

Comment:
Can we make the column names more descriptive:

+ false_positive_rate
+ true_positive_rate
+ partition
+ feature_set